docs/install_shadow.md

```diff
@@ -5,12 +5,17 @@ After building and testing Shadow, the install step is optional. If you do not w
 ```bash
 git clone https://github.com/shadow/shadow.git
 cd shadow
-./setup build --clean --test
-./setup test
+./setup build --clean --test -j`nproc`
+# Optional, but recommended to validate your build and environment.
+./setup test -j`nproc`
+# Optionally install (to ~/.local/bin by default). Can otherwise run the binary
+# directly at build/src/main/shadow.
```

Comment:
I don't think we need the `-j$(nproc)` since the setup script should do this automatically:

https://github.com/shadow/shadow/blob/601c10933ec67f1d8bd1f7c18f91533be1a35709/setup#L130-L135

------

Also since Shadow is sensitive to things like container environments and possibly kernel versions (since we don't test that in our CI), I think it's good to not list the testing step as optional. If people don't want to run it then they won't regardless of what we say, but I think maybe people will run it more often if we don't say anything here.